tools/schema_registry_tools/schemas_command_generator.py

```diff
@@ -34,6 +33,88 @@
 import json
 from pprint import pprint
 import re
+import types
+
+
+def generate_commands(j, test=False):
+    # Extracting Key-Value pairs
+    expected_offset = 0
+    cmd = ''
+    if test:
+        new_offset = []
+    for item in j:
+        if test:
+            new_offset.append(expected_offset)
```

Comment:
I'm certain if this way is not that neat. @BenPope would know more elegant way.

Reply:
It could probably be a bit neater if the records were built as json, and returned from a function called, e.g., `generate_records` - the the test could check that the json is as expected, and the non-test command could build the commands from the json.

Reply:
Good idea. Fixed in 2429790.